### ISSUE TITLE
.init File (name to be challenged)

### ISSUE THREAD

**Issue by baloomba** [thetribeio/generator-project#345]
To avoid a form as long as the one you use to enter the US territory, I think a .init file could be used instead.

The file doesn't exist : The script works as it is now.
I have the file : The file is read and the script ask only the questions for the missing fields.

**Comment by jvasseur** [comment]
We could potentially use the [`.yo-rc.json`](https://yeoman.io/authoring/storage.html) file for this, to prevent having to code the feature from scratch but not sure it's the best file format for this feature.

**Comment by jvasseur** [comment]
This will be partially fixed by #634 expect that for values that are present in the file, they will be used as default value for the answer instead of skipping the question.

This could be fixed by adding a `--skip-answered` option but I'm sure if it is worth it since you can just hit enter to pass the questions.

For information, the file look like this:
```json
{
  "@thetribe/generator-project": {
    "projectName": "test",
    "repositoryName": "thetribeio/test",
    "backend": "express",
    "frontend": "create-react-app",
    "admin": true,
    "domain": "test.thestaging.io",
    "contactEmail": "test@example.com"
  }
}
```
